### ISSUE TITLE
Contact diary: upon entering new contact or place immediately request exposure details entry

### ISSUE THREAD

**Issue by thowa1** [corona-warn-app/cwa-wishlist#775]
When adding new cotacts or places the new item will be added, but the user must then continue finding the new entry and add data such as time, duration and type of exposure.

This is not intuitive and bothersome. I and surely others often enter a new contact for a day and then forget to go back and enter further exposure details.

Requested enhancement:
The exposure details form should be available already when entering new contact and place data without having to search the new contact in the list first.

## Expected Benefits
More complete entries in contact journal.

(Edited for spelling mistakes)

---
Internal Tracking ID: [EXPOSUREAPP-12076](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12076)

**Comment by Ein-Tim** [comment]
Closely related to #734

**Comment by thowa1** [comment]
Agree, enhancement proposed in #734 is a possible solution to the enhancement requested here.

**Comment by larswmh** [comment]
Thanks for your enhancement request @thowa1. We have created an internal ticket for it and will raise this topic internally.
Internal Tracking ID: [EXPOSUREAPP-12076](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12076)

---
Corona-Warn-App Open Source Team
